### ISSUE TITLE
Doc changes are failing silently

### ISSUE THREAD

**Issue by staycoolcall911** [winglang/wing#5168]
### I tried this:

Recently tried to update the docs, but learned the automatic "update docs" PR was failing for 4 days without us noticing it.
See an example: 
https://github.com/winglang/docsite/pull/708
[winglang/docsite/actions/runs/7034089517/job/19141315335?pr=708](https://github.com/winglang/docsite/actions/runs/7034089517/job/19141315335?pr=708)

### This happened:

The "update docs" PR was silently failing.

### I expected this:

A monitor to alert the current on-call team that the "update docs" PR failed.

### Is there a workaround?

Never sleep, always hit the refresh button looking for this failed PR.

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by ainvoner** [comment]
@staycoolcall911 perhaps we can add a slack alert whenever those specific PRs fail

**Comment by staycoolcall911** [comment]
Yes, though my original thought was we can add a monitor to BetterStack, so it will notify on-call out of the box
